### PR TITLE
parallel-letter-frequency: remove characters with accents / diacritics

### LINKED ISSUE #1104: parallel-letter-frequency: illegal character error for test file
Testing a basic stub implementation of this exercise (on Windows with VS Code) results in three errors.

```java
// Stub implementation - first step of TDD
import java.util.Map;

class ParallelLetterFrequency {
    ParallelLetterFrequency(String input) {
        throw new UnsupportedOperationException();
    }

    Map<Integer, Integer> letterCounts() {
        throw new UnsupportedOperationException();
    }
}
```

```bash
# Errors from build output
ParallelLetterFrequencyTest.java:146: error: unclosed character literal
        assertEquals(new Integer(2), p.letterCounts().get((int) '├╝'));
                                                                ^
ParallelLetterFrequencyTest.java:146: error: illegal character: '\u00bc'
        assertEquals(new Integer(2), p.letterCounts().get((int) '├╝'));
                                                                  ^
ParallelLetterFrequencyTest.java:146: error: unclosed character literal
        assertEquals(new Integer(2), p.letterCounts().get((int) '├╝'));
```

Changing the literal at line 146 of the test file from `'ü'` to `'\u00FC'` fixes this - not sure if that is cross platform or not though? If it is, this issue is probably a candidate for <kbd>[good-first-patch]()</kbd>
```java
        assertEquals(new Integer(2), p.letterCounts().get((int) '\u00FC')); //Unicode ü
```

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Fixes #1104.

I have removed the two Strings called 'OdeAnDieFreude' and Wilhelmus. The test case for appending all three anthems has been removed, and in the comma test case, I have substituted 'StarSpangledBannner'.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by sjwarner-bp** [comment]
As per discussion on #1104, I'd like to know what others think of this change, and if I have reduced test coverage too much. I think it should be ok, however.

**Comment by sjwarner-bp** [comment]
Thanks @FridaTveit, glad you agree :slightly_smiling_face: :christmas_tree:

File: exercises/parallel-letter-frequency/.meta/HINTS.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Single-threaded (non-concurrent) solutions can pass all tests [but the last.](https://www.youtube.com/watch?v=mJZZNHekEQw)  Your solution will be tested for concurrency by submitting it as a [Runnable](https://docs.oracle.com/javase/7/docs/api/java/lang/Runnable.html) to an [ExecutorService.](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ExecutorService.html) Your solution must leverage multiple [Threads](https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html) to pass the final test.
 
 Java documentation on [parallel streams](https://docs.oracle.com/javase/tutorial/collections/streams/parallelism.html) may provide some help.
+
+As a stretch goal, consider if your implementation will work for characters with [diacritics or accents](https://en.wikipedia.org/wiki/Diacritic). For example, a good solution should not consider e and ë the same character. An example text for this case is [Wilhelmus](https://en.wikipedia.org/wiki/Wilhelmus), the dutch national anthem.
```
Comment (stkent) [inline exercises/parallel-letter-frequency/.meta/HINTS.md]: I'm a bit wary of using the phrase `a good solution` here; maybe just `such solutions`?

Comment (stkent) [inline exercises/parallel-letter-frequency/.meta/HINTS.md]: nit: `dutch` -> `Dutch`.

**Comment by stkent** [comment]
Couple minor comments, LGTM otherwise!

File: exercises/parallel-letter-frequency/.meta/HINTS.md (hunk 1 above)
Comment (sjwarner-bp) [reply exercises/parallel-letter-frequency/.meta/HINTS.md]: Good spot :slightly_smiling_face:

**Pull request merged**